### PR TITLE
Isolated declarations performance improvements

### PR DESCRIPTION
**Pull request by dragomirtitian**

### CODE REVIEW

File: src/testRunner/compilerRunner.ts [hunk 1]
```diff
@@ -133,13 +135,16 @@ export class CompilerBaselineRunner extends RunnerBase {
...
+            before(function () {
                 const fixedIsolatedTestEnv = FixedIsolatedDeclarationTest.fixTestProject(environment);
                 if (fixedIsolatedTestEnv) {
                     fixedIsolatedTest = new FixedIsolatedDeclarationTest(fixedIsolatedTestEnv);
+                } else {
+                    this.skip();
                 }
             });
             it(`Correct dte emit for ${fileName}`, () => fixedIsolatedTest?.verifyDteOutput());
+            it(`Correct dte emit for ${fileName}`, () => fixedIsolatedTest?.verifyDteOutput());
```
Comment (h-joo) [inline src/testRunner/compilerRunner.ts]: Was there a reason this line is duplicated? If so, can you comment why?

File: src/harness/isolatedDeclarationFixer.ts [hunk 2]
```diff
@@ -80,6 +80,13 @@ export function fixProjectInternal(
             let diagnostics = getIsolatedDeclarationsErrors(file.fileName);
 
             if (diagnostics.length === 0) continue;
+
+            // Try to fix all
```
Comment (h-joo) [inline src/harness/isolatedDeclarationFixer.ts]: Nit: Does this comment add value to the reader?

**Review by h-joo — APPROVED** [review]
LGTM with two comments

File: src/testRunner/compilerRunner.ts (hunk 1 above)
Comment (dragomirtitian) [reply src/testRunner/compilerRunner.ts]: Thanks for catching this one. Removed.

File: src/harness/isolatedDeclarationFixer.ts (hunk 2 above)
Comment (dragomirtitian) [reply src/harness/isolatedDeclarationFixer.ts]: Removed.

**Pull request merged**